app/forms/resend_email_confirmation_form.rb

```diff
@@ -3,14 +3,15 @@
 class ResendEmailConfirmationForm
   include ActiveModel::Model
 
-  attr_reader :email
+  attr_reader :email, :terms_accepted
```

Comment:
what if we did a predicate getter method?

```suggestion
  attr_reader :email

  def terms_accepted?
    @terms_accepted
  end
```

Reply:
This relies on using SimpleForm to map `terms_accepted` into the field value, which I'm not 100% sure works with predicate method, but I'll test to see if it works.